CONTRIBUTING.md

```diff
@@ -0,0 +1,41 @@
+# Contributing and Maintaining
+
+First, thank you for taking the time to contribute!
+
+The following is a set of guidelines for contributors as well as information and instructions around our maintenance process. The two are closely tied together in terms of how we all work together and set expectations, so while you may not need to know everything in here to submit an issue or pull request, it's best to keep them in the same document.
+
+## Ways to contribute
+
+Contributing isn't just writing code - it's anything that improves the project. All contributions for Restricted Site Access are managed right here on GitHub. Here are some ways you can help:
+
+### Reporting bugs
+
+If you're running into an issue with the plugin, please take a look through [existing issues](https://github.com/10up/restricted-site-access/issues) and [open a new one](https://github.com/10up/restricted-site-access/issues/new) if needed. If you're able, include steps to reproduce, environment information, and screenshots/screencasts as relevant.
+
+### Suggesting enhancements
+
+New features and enhancements are also managed via [issues](https://github.com/10up/restricted-site-access/issues).
+
+### Pull requests
+
+Pull requests represent a proposed solution to a specified problem. They should always reference an issue that describes the problem and contains discussion about the problem itself. Discussion on pull requests should be limited to the pull request itself, i.e. code review.
+
+For more on how 10up writes and manages code, check out our [10up Engineering Best Practices](https://10up.github.io/Engineering-Best-Practices/).
+
+## Workflow
+
+The `develop` branch is the development branch which means it contains the next version to be released. `stable` contains the current latest release and `master` contains the corresponding stable development version. Always work on the `develop` branch and open up PRs against `develop`.
+
+## Release instructions
```

Comment:
These aren't really relevant to this plugin - I would suggest we copy from `ads-txt` instead for now and can revisit once we get GH Actions in place.

Reply:
Done, see: https://github.com/10up/restricted-site-access/pull/73/commits/7b13d0b4a41a823d0a16871945fc1621b2dc452e